### DISCUSSION TITLE
whatsapp monitor cmd

### DISCUSSION THREAD

**Discussion by lmonkeybomb** [rizwansoaib/whatsapp-monitor#184] [Q&A]
on cmd if i select on whatsapp monitor open whatsapp web it says on cmd cannot find chrome library

**Comment by rizwansoaib** [comment]
Why you need cmd now 🧐

**Reply by lmonkeybomb** [reply]
Because on normal monitor i have to keep it open but i want a monitor that works when i do other stuff on pc is it possible? i thought using cmd feature it could work like that

**Comment by rizwansoaib** [comment]
#145
